src/compiler/scala/tools/nsc/ast/Trees.scala

```diff
@@ -41,8 +41,6 @@ trait Trees extends scala.reflect.internal.Trees { self: Global =>
   case class InjectDerivedValue(arg: Tree)
        extends SymTree with TermTree
 
-  class PostfixSelect(qual: Tree, name: Name) extends Select(qual, name)
```

Comment:
I searched GitHub for `PostfixSelect`, found this one: https://github.com/scalameta/trees/blob/fb4cbb515b9a4b65fd679f1e9c13860a7038b748/semanticdb/scalac/library/src/main/scala-2.12.7/scala/meta/internal/semanticdb/scalac/SemanticdbAnalyzer.scala#L539

@gabro / @tgodzik is this code still relevant?

Reply:
Looks like it was inlined into scalameta/scalameta and only used for versions prior to 2.12.8, which is no longer supported. So we should be good I think.